repo/fsrepo/fsrepo.go

```diff
@@ -557,6 +565,27 @@ func (r *FSRepo) Datastore() repo.Datastore {
 	return d
 }
 
+func (r *FSRepo) DirectMount(prefix string) ds.Datastore {
```

Comment:
Maybe call this `DatastoreAt` or `GetMount`. `DirectMount` just leaves me confused about what this is